app/models/expanded_links.rb

```diff
@@ -0,0 +1,27 @@
+class ExpandedLinks < ApplicationRecord
+  self.table_name = "expanded_links"
```

Comment:
I thought I did for Rails pluralising the class name (which seemed rubbish as ExpandedLink) but seems it works fine without this so I'll drop it.